README.md

```diff
@@ -24,9 +60,58 @@ With blockchain we can move financial services to a new level - Decentralized Fi
+### How
+
+We build `NEAR-CLP` to fulfill this goals in a very open and decentralized manner. Here are our **GOALS**:
 
 - Focus on liquidity pools and AMM
-- Eliminate side markets incentives on NEARswap. Side market is an entity which could change a behavior of the CLP / AMM protocol and shift the benefits or potentially manipulate the whole market.
-- Highly predictable behavior designed with the main blockchain principles: trustless smart-contracts.
+- Eliminate side markets incentives. Side market is an entity which could change a behavior of the CLP / AMM protocol and shift the benefits or potentially manipulate the whole market.
+- Highly predictable behavior designed with the main blockchain principles: trustless smart-contracts without susceptible governance.
+
+
+## Why NEAR
+
+Thriving DeFi ecosystem needs:
+
+1. Robust layer-1 to enable composability between different contracts (→ DeFi Lego).
+2. Secure platform with adjustable throughput and storage capabilities. With DeFi we can't take a risk of unreliable or corruptible layer-2.
+
+NEAR is providing a solution for above requirements today. With it's unique set of features and scaling potential aspires to be a significant player in the new Decentralize Finance world.
+
+
+### Why Bridging with Ethereum?
+
+Today, most of the valuable assets are hosted on Ethereum. In fact, today, DeFi is an Ethereum domain and it has the biggest ecosystem (developers, tools, dapps). It's also very secure. However Ethereum has few very important pain points:
+
+- it doesn't scale
+- layer-2 solutions are complex and not composable
+- EVM execution model is unsafe
+
+We want to enable people to use Ethereum based assets and do asset management on NEAR using NEAR growing ecosystem. Through the inter-blockchain **bridge**, NEAR will provide a scale-ability to the current Ethereum DeFi ecosystem and more robust smart-contract runtime.
+
+
+
+## License
+
+We create foundations for DeFi, hence an open access is the key.
+
+We build the Open Source software and knowledge base. All our work is licensed with a Mozilla Public License (MPL) version 2.0. Refer to [LICENSE](LICENSE) for terms.
+
+The MPL is a simple copyleft license. The MPL's "file-level" copyleft is designed to encourage contributors to share modifications they make to your code, while still allowing them to combine your code with code under other licenses (open or proprietary) with minimal restrictions.
+Q2: Why yet another open source license?
+
+The MPL fills a useful space in the spectrum of free and open source software licenses, sitting between the Apache license, which does not require modifications to be shared, and the GNU family of licenses, which requires modifications to be shared under a much broader set of circumstances than the MPL.
+
+[FAQ](https://www.mozilla.org/en-US/MPL/2.0/FAQ/).
+
+
+
+## Contact
+
++ Leave a note using github issues.
++ [Robert Zaremba](https://zaremba.ch/contact.html)
+
```

Comment:
Please add a mention of the "Hack the Rainbow" hackathon, the Team, and the names/github links of all four of us.

Reply:
done